### PR TITLE
add remote and disableremoteplayback attrs

### PR DESCRIPTION
**Pull request by siusin**
To address #1047.

Test available as: https://w3c.github.io/test-results/remote-playback/all.html

### CODE REVIEW

**Comment by LJWatson** [comment]
LGTM.

File: sections/semantics-embedded-content.include [hunk 1]
```diff
@@ -7963,6 +7973,17 @@ zero or more <{track}> elements, then
...
   <hr />
 
+  The <dfn element-attr for="media"><code>remote</code></dfn> attribute MUST return the <a>RemotePlayback</a> instance associated with the <a lt="media elements">media element</a>.
+
+  The <dfn element-attr for="media"><code>disableRemotePlayback</code></dfn> attribute is a <a>boolean
+  attribute</a>. When present, the user agent MUST NOT play the <a lt="media elements">media element</a> remotely or present any UI to do so.
+
+  When the <{media/disableRemotePlayback}> attribute is added to the <a lt="media elements">media element</a>, the user agent MUST run the steps to <a>disable remote playback</a>.
+
+  A corresponding <dfn attribute for="HTMLMediaElement"><code>disableRemotePlayback</code></dfn> IDL attribute which reflects the value of each element’s <{media/disableRemotePlayback}> content attribute is added to the {{HTMLMediaElement}} interface. The {{HTMLMediaElement/disableRemotePlayback}} IDL attribute MUST <a>reflect</a> the content attribute of the same name.
```
Comment (chaals) [inline sections/semantics-embedded-content.include]: please put a line break in here somewhere

**Review by chaals — CHANGES_REQUESTED** [review]
Mostly LGTM. A couple of administrivia tweaks and it's ready to merge.

please update acknowledgements - whether individual or in the bit about WGs whose work we are incorporating.

**Comment by siusin** [comment]
Thanks @chaals and @LJWatson . PR updated.

**Review by chaals — APPROVED** [review]

**Pull request merged**